test/Feature/UpdateSpec.hs

```diff
@@ -51,32 +51,14 @@ spec = do
 
     context "in a nonempty table" $ do
       it "can update a single item" $ do
-        get "/items?id=eq.42"
-          `shouldRespondWith`
-            [json|[]|]
-
-        request methodPatch "/items?id=eq.2"
-            [("Prefer", "tx=commit")]
```

Comment:
Persistance of PATCH is tested in the rollback specs. Here it just leads to more concurrency issues, so I decided to take it out as well. Had already done that for the INSERT on the "all tests rollback" PR, I think.

Reply:
Yes, looks good. I see PATCH persistence is already here:

https://github.com/PostgREST/postgrest/blob/a52c114ff43c4576d491106df11177bcb8a8cdbb/test/Feature/RollbackSpec.hs#L100-L114